artemis-journal/src/main/java/org/apache/activemq/artemis/core/journal/impl/JournalRecord.java

```diff
@@ -45,26 +44,51 @@ public JournalRecord(final JournalFile addFile, final int size) {
       addFile.addSize(size);
    }
 
-   void addUpdateFile(final JournalFile updateFile, final int size) {
-      if (updateFiles == null) {
-         updateFiles = new ArrayList<>();
+   private void checkNotDeleted() {
+      if (fileUpdates == DELETED) {
```

Comment:
I made a small ammend to this while I merged. I moved

I moved checkNotDeleted to the end of the file so the diffs are clearer.. method by method.


you dont' have to do anything.. just letting you know.

Reply:
@franz1981  ^